docs/en/Running-Spark-on-Alluxio.md

```diff
@@ -200,17 +200,18 @@ cluster with `--num-executors=1`, Spark might place the only executor on `host2`
 However, if `--num-executors=2` and executors are started on `host1` and `host2`, Spark will be smart
 enough to prioritize placing the job on `host1`.
 
-## `Failed to login` Issues with Spark Shell
+## `Class alluxio.hadoop.FileSystem not found` Issues with SparkSQL
 
 To run the `spark-shell` with the Alluxio client, the Alluxio client jar will have to be added to the classpath of the
-Spark driver and Spark executors, as described earlier. However, sometimes Alluxio will fail to determine the security
-user and will result in an error message similar to: `Failed to login: No Alluxio User is found.` Here are some
-solutions.
+Spark driver and Spark executors, as [described earlier](Running-Spark-on-Alluxio.html#general-setup). 
+However, sometimes SparkSQL fail to save tables to Hive metastore when the table locations in in Alluxio with an error
```

Comment:
how about something like:

"... SparkSQL may fail to save tables to the Hive metastore (location in Alluxio), with an error ..."

Reply:
fixed